backend/src/main/twirl/MainFutureNettyScala3.scala.txt

```diff
@@ -1,18 +1,22 @@
-@(groupId: String, addDocumentation: Boolean)
+@(groupId: String, addDocumentation: Boolean, addMetrics: Boolean)
 package @groupId
 
-import sttp.tapir.server.netty.NettyFutureServer
+import sttp.tapir.server.netty.@if(addMetrics){{NettyFutureServer, NettyFutureServerOptions}}else{NettyFutureServer}
 
 import scala.concurrent.duration.Duration
 import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.io.StdIn
 import ExecutionContext.Implicits.global
 
 @@main def run(): Unit =
+  @if(addMetrics) {
+  val serverOptions = NettyFutureServerOptions.default
+    .appendInterceptor(Endpoints.prometheusMetrics.metricsInterceptor())
```

Comment:
`CustomiseInterceptors` has a `.metricsInterceptor` method which adds the interceptor in the correct order (should be prepended, not appended)

Reply:
Thanks 👍  updated for all cases.